spec/requests/querying_spec.rb

```diff
@@ -116,14 +116,6 @@ def create_vms_by_name(names)
       expect(response.parsed_body["links"].keys).to match_array(%w(self previous first last))
     end
 
-    it "only returns paging links if both offset and limit are specified" do
-      create_vms_by_name %w(aa bb)
-
-      get api_vms_url, :params => { :offset => 0, :expand => :resources }
-
-      expect(response.parsed_body.keys).to eq(%w(name count subcount resources actions))
-    end
-
```

Comment:
maybe replace with a simple query returns self/first/last.  (next/previous won't be there for a small data set).